org.palladiosimulator.analyzer.slingshot.behavior.spd/src/org/palladiosimulator/analyzer/slingshot/behavior/spd/interpreter/entity/trigger/AbstractManagedElementTriggerChecker.java

```diff
@@ -101,7 +106,6 @@ protected FilterResult getResult(final DESEvent event) {
 		}
 		
 		final double aggregatedValue = this.aggregator.getCurrentValue();
-		System.out.println("Aggregated value, "+aggregatedValue);
 		if (this.compareToTrigger(aggregatedValue) == ComparatorResult.IN_ACCORDANCE) {
 			return FilterResult.success(event);
 		}
```

Comment:
While we are already at it, how about giving the disregard in line 112 a reason? Something à la "Value and Expectation not in accordance?"